selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -1393,4 +1393,9 @@ public void waitSuccessCondition(ExpectedCondition<Boolean> expression, int time
   public void waitSuccessCondition(ExpectedCondition<Boolean> expression) {
     waitSuccessCondition(expression, DEFAULT_TIMEOUT);
   }
+
+  public void closeCurrentWinAndReturnToMainTab(String mainTabHandle) {
```

Comment:
`MainTab` is confusing name a little. In fact, it is switching to any window by its name or handle. Also, the method doesn't _return to_ some tab - it is _switching to_ it.
Therefore, next signature will look more appropriate, IMHO: `closeCurrentWindowAndSwitchToAnotherOne(String windowNameOrHandle)`.